frontend/src/store/modules/taste/getters.js

```diff
@@ -6,34 +6,11 @@ export default {
             return [];
         }
     },
-    getCustomTastesIds: state => {
-        if (state.customTastes.byId !== undefined) {
-            return Object.keys(state.customTastes.byId);
-        } else {
-            return [];
-        }
-    },
     getAllTastesIds: state => {
```

Comment:
It is not related to your task, but it is better in the state with `byId` also save an array `allIds`, then this method would be unnecessary. Just remark, it is not required to fix it for now.